pkgs/applications/misc/furtherance/default.nix

```diff
@@ -4,21 +4,21 @@
 , dbus, gtk4, sqlite
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
I just noticed that you're not using `rustPlatform.buildRustPackage`, is there a reason for that?

Reply:
Here's an example of derivation using it: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/typesetting/typst/default.nix